_posts/fr/newsletters/2023-10-04-newsletter.md

```diff
@@ -0,0 +1,134 @@
+---
+titre: 'Bulletin Hebdomadaire Bitcoin Optech #271'
```

Comment:
I noticed that the title of this newsletter was just "Newsletter". It turns out that the title’s attribute name got translated:

```suggestion
title: 'Bulletin Hebdomadaire Bitcoin Optech #271'
```